2019/03/16 14:35:00 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/03/16 14:35:30 [error] 6#6: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/03/16 14:47:10 [error] 6#6: *25 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
